src/menu/types.js

```diff
@@ -122,6 +123,7 @@ export type StatefulContainerPropsT = {
   onItemSelect: OnItemSelectFnT,
   /** Ref for the menu container element. Used to capture key events for navigation */
   rootRef?: RootRefT,
+  listenRef: listenRefT,
```

Comment:
Can you please change `listenRef` throughout the PR to `controlNode` or `keyboardControlNode`? I'd like the label to convey that it doesn't listen to every event, just key events and this is not a ref box, but rather the unboxed dom node